packages/mobile/capacitor/plugins/secure-filesystem-access/android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.iota.plugins.securefilesystemaccess">
+     
+  <!-- Permissions -->
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Endup using:
`<uses-permission
        android:name="android.permission.WRITE_EXTERNAL_STORAGE"
        tools:ignore="ScopedStorage" />`
I've removed from here because it's needed on the main app project folder.
We don't need the premissions for Android 11+ only as it's using the new Scoped storage, but we need for versions 5-10. In the plugin we will need to implement import old files to new way, this would be ready for beta I think.
Also we are using on the main app manifest `requestLegacyExternalStorage=true` to force Android10 also to use old storage, since was the only way I get it working.